projects/components/src/stepper/tab/stepper-tab.component.ts

```diff
@@ -34,4 +35,11 @@ export class StepperTabComponent extends ContentHolder {
 
   @Input()
   public actionButtonLabel?: string;
+
+  private readonly refreshSubject: BehaviorSubject<void> = new BehaviorSubject<void>(undefined);
+  public readonly refresh$: Observable<void> = this.refreshSubject.asObservable();
+
+  public ngOnChanges(): void {
+    this.refreshSubject.next();
```

Comment:
I see that there are many other properties in this component. Should we only emit when` actionButtonLabel` change? We could probably rename `refresh` to be more specific as well.

Reply:
It's highly likely that this issue could be there for other inputs as well. But I've only handled it for actoon button label for noe.